pants.remote-execution.toml

```diff
@@ -17,4 +17,4 @@ remote_execution_address = "grpc://127.0.0.1:50051"
 remote_instance_name = ""
 
 # TODO: See https://gitlab.com/BuildGrid/buildgrid/-/blob/master/buildgrid/server/server.py#L212-222
-process_execution_remote_parallelism = 8
+process_execution_remote_parallelism = 4
```

Comment:
Related?

Reply:
Somewhat... because the server is running in Docker and choosing its worker count based on the host machine's CPU count, this will make using the server less likely to encounter the error referenced in the TODO (unless they happen to be running on a very small machine).